tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -67,11 +67,23 @@
 
 tf.flags.DEFINE_float('momentum', 0.9, 'Momentum for MomentumOptimizer.')
 
-tf.flags.DEFINE_float('weight_decay', 1e-4, 'Weight decay for convolutions.')
+tf.flags.DEFINE_float('weight_decay', 2e-4, 'Weight decay for convolutions.')
 
 tf.flags.DEFINE_boolean('use_distortion_for_training', True,
                         'If doing image distortion for training.')
 
+tf.flags.DEFINE_boolean('run_experiment', False,
```

Comment:
run_experiment doesn't tell much to users new to TF. how's experiment different from estimator? how does experiment go with distributed TF? If running experiment give superset info than Estimator alone, why should we keep estimator option alive?